MFVNeutralino/plugins/AnalysisCuts.cc

```diff
@@ -206,6 +206,28 @@ bool MFVAnalysisCuts::filter(edm::Event& event, const edm::EventSetup&) {
       if(!success) return false;
     }
 
+    // MET trigger
+
+    if (apply_presel == 5) {
+
+      // Veto events which pass HT trigger and offline HT > 1200 GeV, to keep orthogonal with apply_presel == 1
+      if(satisfiesTrigger(mevent, mfv::b_HLT_PFHT1050)) return false;
+
+      if ( !satisfiesTrigger(mevent, mfv::b_HLT_PFMET120_PFMHT120_IDTight) ){
+        return false;
+      }
+    }
+
+    // Events not passing MET trigger ans have MET lower than threshold
+    if (apply_presel == 6) {
+      if(satisfiesTrigger(mevent, mfv::b_HLT_PFHT1050)) return false;
+
+      //if ( !satisfiesTrigger(mevent, mfv::b_HLT_PFMET120_PFMHT120_IDTight) ){
+      //  std::cout << "MET trigger not fired" << std::endl;
+      //}
+      if (mevent->met()>=150) return false;
+    }
```

Comment:
Did you mean for this to have a MET trigger requirement?

Reply:
The apply_presel=5 is actually the MET trigger pre-selection. This one previously aims to select events with low MET for ML training purposes and not used now.

Reply:
Ah, maybe the comment just needs to be updated?

`    // Events not passing MET trigger ans have MET lower than threshold
`